metadata-service/schema-registry-api/generated/src/main/java/io/datahubproject/schema_registry/openapi/generated/Config.java

```diff
@@ -0,0 +1,122 @@
+package io.datahubproject.schema_registry.openapi.generated;
+
+import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import io.swagger.v3.oas.annotations.media.Schema;
+import org.springframework.validation.annotation.Validated;
+import javax.validation.Valid;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import javax.validation.constraints.*;
+
+/**
+ * Config
+ */
+@io.swagger.v3.oas.annotations.media.Schema(description = "Config")
+@Validated
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2022-12-20T16:52:36.517693Z[Europe/Lisbon]")
```

Comment:
Are these raw generated classes? The modified Controllers make sense to commit, but if these are raw I'm not sure they do.

Reply:
These are modified, only the controllers are not. I committed all to git because the swagger plugin does not allow us generate the interfaces but not the controllers. I could conceivably create a moustache template that explicitly does not render controllers but that seemed a non-trivial amount of work for the upside.

Reply:
Fair enough 😄